airbyte-webapp/src/packages/cloud/views/credits/CreditsPage/CreditsPage.tsx

```diff
@@ -18,6 +21,10 @@ import styles from "./CreditsPage.module.scss";
 const CreditsPage: React.FC = () => {
   const { emailVerified } = useAuthService();
   useTrackPage(PageTrackingCodes.CREDITS);
+  const { data } = useFreeConnectorProgramInfo();
+  const isFreeConnectorProgramEnabled = useExperiment("workspace.freeConnectorsProgram.visible", false);
+  const showEnrollmentUi = isFreeConnectorProgramEnabled && data?.showEnrollmentUi;
```

Comment:
As mentioned on slack, I wonder if we can just roll the `useExperiment()` call into `useFreeConnectorProgramInfo()` so that `data.showEnrollmentUi = false` if the experiment is disabled.